skops/io/tests/test_persist.py

```diff
@@ -880,3 +881,63 @@ def test_estimator_with_bytes_files_created(tmp_path):
         files = input_zip.namelist()
     bin_files = [file for file in files if file.endswith(".bin")]
     assert len(bin_files) == 2
+
+
+OPERATORS = [
+    ("add", partial(operator.add, 0)),
+    ("sub", partial(operator.sub, 0)),
+    ("mul", partial(operator.mul, 1)),
+    ("truediv", partial(operator.truediv, 1)),
+    ("pow", partial(operator.pow, 1)),
+    ("matmul", partial(operator.matmul, np.eye(N_SAMPLES))),
+    ("ge", partial(operator.ge, 0)),
+    ("gt", partial(operator.gt, 0)),
+    ("le", partial(operator.le, 0)),
+    ("lt", partial(operator.lt, 0)),
+    ("eq", partial(operator.eq, 0)),
+    ("neg", operator.neg),
+    ("attrgetter", operator.attrgetter("real")),
+    ("attrgetter", operator.attrgetter("real", "real")),
+    ("itemgetter", operator.itemgetter(None)),
+    ("itemgetter", operator.itemgetter(None, None)),
+    ("methodcaller", operator.methodcaller("round")),
+    ("methodcaller", operator.methodcaller("round", 2)),
```

Comment:
could we add `delitem`, `setitem`, `call`, and an inplace one like `iadd`?

Reply:
The problem with `delitem` and `setitem` is that they can't be really used in this context. They need to be called with the collection as first argument, so when I set them as `partial(operator.delitem, 0)`, for instance, it won't work when this is called with `X` (it tries to delete `X` from 0 instead of the other way round). And since these operators take no keyword arguments, it's not possible to set up `partial` to call the arguments in the right order.

I did, however, add all the `i*` operators for the functions already tested, and for Python >= 3.11, I added `operator.call`.